### PR TITLE
feat: add type as first class attribute in relationship definition

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
@leecalcote As this gets merged we should make a release with v1beta1 tag?

**Comment by leecalcote** [comment]
@MUzairS15 yes, please.

**Comment by leecalcote** [comment]
There's an open PR that deletes the failing (and inappropriate) eslint-check / runner / eslint (pull_request) workflow.

File: models/relationship.json [hunk 1]
```diff
@@ -33,6 +33,10 @@
       "$ref": "https://schemas.meshery.io/model.json",
       "description": "Model of the Relationship."
     },
+    "type": {
+      "type": "string",
+      "description": "Used for classification of Relationships. Groups the relationships having similar behaviour."
```
Comment (leecalcote) [inline models/relationship.json:38]:
```suggestion
      "description": "Classification of relationships. Used to group relationships similar in nature."
```

**Comment by MUzairS15** [comment]
That's merged.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
LGTM

**Pull request merged**